pages/common/wp.md

```diff
@@ -0,0 +1,24 @@
+# wp
+
+> The official command-line interface to manage WordPress instances.
+> More information: <https://wp-cli.org/>.
+
+- Describe the operating system, shell, PHP, and WP-CLI installation:
```

Comment:
```suggestion
- Describe the operating system, shell, PHP and WP-CLI installation:
```

Small error I missed

Reply:
Hmm, that's not an error, that's an [Oxford comma](https://en.wikipedia.org/wiki/Serial_comma).

There are many examples of it throughout this repository.

![image](https://user-images.githubusercontent.com/22801583/118687128-f4b7d380-b804-11eb-905c-11b0f304f6a1.png)

Reply:
Ah, interesting! I think we can keep it.

By the way @SethFalco did you know that in regex `\w` = `[a-zA-Z0-9$_]`? Using `\w` would shorten that regex.

Reply:
Ahh, nice! Thanks for the heads up. \\(^-^)/